### DISCUSSION TITLE
Help with cape-dabbrev and custom angular (html) mode.

### DISCUSSION THREAD

**Discussion by JordanAnthonyKing** [minad/cape#140] [Q&A]
Hi, I've been messing around with creating a mode for angular and using dabbrev for completion. It works almost entirely but I don't get completions for custom html tags, i.e. symbols following a `<`. If I just start typing the name of the tag, i.e. `my-custom-tag` then dabbrev suggestions are shown, but trying `<my-custom-tag` doesn't work.

After some messing around I thought this might be due to my syntax-table for angular, but even after changing <> from brackets to whitespace it still doesn't work:
```
(defvar angular-syntax-table 
  (let ((table (make-syntax-table text-mode-syntax-table)))
    ;; (modify-syntax-entry ?< "(>" table)
    ;; (modify-syntax-entry ?> ")<" table)
    (modify-syntax-entry ?< " " table)
    (modify-syntax-entry ?> " " table)
    (modify-syntax-entry ?: "_" table)
    (modify-syntax-entry ?_ "_" table)
    (modify-syntax-entry ?. " " table) table))
```

I've tried messing around with a few other things too but no luck.
Any advice?

**Comment by minad** [comment]
I don't have a good suggestion. I would avoid modifying the syntax table since this will affect many other features of the mode.
